sdk/Sdk.Generators/FunctionMetadataProviderGenerator.Emitter.cs

```diff
@@ -122,18 +123,31 @@ private void AddBindingInfo(IndentedTextWriter indentedTextWriter, string functi
                 foreach (var binding in bindings)
                 {
                     var bindingVarName = functionVarName + "binding" + bindingCount.ToString();
-                    indentedTextWriter.WriteLine($"var {bindingVarName} = new {{");
-                    indentedTextWriter.Indent++;
-                    
+                    StringBuilder jsonStr = new StringBuilder("$\"{{");
```

Comment:
@satvu is there a reason we're building the JSON string by hand here? We should be able to use JSON serialization in the context of the source generator, but persist the JSON string in the generated code.

Reply:
Let me check if I can make that work with the AuthorizationLevel - we can't parse it to the correct value until runtime b/c it depends on the HTTP Extensions package which we won't have access to.